### ISSUE TITLE
(CCK 3.10) Auto Generation Option loss

### ISSUE THREAD

**Issue by JOELwindows7** [NotAKidOnSteam/SimpleAAS#8]
# Auto Generation Option loss
The NAK Modular Setting did notr store Auto Generation Options available on some type of the AAS. Such as Color, Toggle, Dropdown, & Slider.

Say I copied my original AAS from `CCKAvatar` into `NAKModularSetting` file. And then using `NAKSimpleAASParameter` component to re-merge along with other AAS modules. If I resync it, my AAS is updated with my original setting along with other modular settings, **except their Auto Generation Options**. Additionally, with some type, their default value did not copied from the AAS to the `NAKSimpleAASParameter` asset due to either CCK bug or this.

Consider adding such features that preserves OR prepare these Auto Generation Options right from `NAKModularSetting` asset file.

## Screenshots
![image](https://github.com/NotAKidoS/SimpleAAS/assets/7281060/4bcd0571-8bab-4b4a-9167-d63a6b9c1b56)
![image](https://github.com/NotAKidoS/SimpleAAS/assets/7281060/a67feea0-afff-4898-bc2d-2533e2c291b8)
![image](https://github.com/NotAKidoS/SimpleAAS/assets/7281060/fa0403da-4a65-48f0-80ff-918c2b36764d)
![image](https://github.com/NotAKidoS/SimpleAAS/assets/7281060/aa401b2f-8eda-4a1e-be1a-c8c4560066e0)
Don't worry, I can just get out of the prefab discarding the change.

**Comment by NotAKidoS** [comment]
My tool is made for those not using the CVRAvatar autogen and as such has been excluded from the tool. The copying from CVRAvatar to the parameter asset not doing a deepcopy is also a CCK bug that is reported on the feedback board.

While I could copy the autogen to the parameters asset, I would not be able to reference the original assets, as the parameter asset is meant to be shared between avatars- meaning the reference to the objects would be wrong.

Basically, this is not in scope of what the tool is meant to do and as such will *probably* not happen. Personally I avoid the autogen completely due to how broken it can be.
